test/integration/dialects/sqlite/connection-manager.test.js

```diff
@@ -32,19 +34,29 @@ if (dialect === 'sqlite') {
           });
         })
         .then(() => {
-          expect(fs.existsSync(path.join(__dirname, fileName))).to.be.true;
-          expect(fs.existsSync(path.join(__dirname, `${fileName}-shm`)), 'shm file should exists').to.be.true;
-          expect(fs.existsSync(path.join(__dirname, `${fileName}-wal`)), 'wal file should exists').to.be.true;
+          expect(jetpack.exists(path.join(__dirname, fileName))).to.be.equal('file');
+          expect(jetpack.exists(path.join(__dirname, `${fileName}-shm`)), 'shm file should exists').to.be.equal('file');
+          expect(jetpack.exists(path.join(__dirname, `${fileName}-wal`)), 'wal file should exists').to.be.equal('file');
 
           return sequelize.close();
         })
         .then(() => {
-          expect(fs.existsSync(path.join(__dirname, fileName))).to.be.true;
-          expect(fs.existsSync(path.join(__dirname, `${fileName}-shm`)), 'shm file exists').to.be.false;
-          expect(fs.existsSync(path.join(__dirname, `${fileName}-wal`)), 'wal file exists').to.be.false;
+          expect(jetpack.exists(path.join(__dirname, fileName))).to.be.equal('file');
+          expect(jetpack.exists(path.join(__dirname, `${fileName}-shm`)), 'shm file exists').to.be.false;
+          expect(jetpack.exists(path.join(__dirname, `${fileName}-wal`)), 'wal file exists').to.be.false;
 
           return this.sequelize.query('PRAGMA journal_mode = DELETE');
         });
     });
+
+    it('automatic path provision for `options.storage`', () => {
+      return Support.createSequelizeInstance({
+        storage: path.join(__dirname, folderName, fileName)
```

Comment:
Also with my suggestion above you can now do

```suggestion
        storage: jetpack.path(folderName, fileName)
```